bin/node/runtime/src/lib.rs

```diff
@@ -1206,15 +1206,15 @@ impl pallet_contracts::Config for Runtime {
 	type CallFilter = Nothing;
 	type DepositPerItem = DepositPerItem;
 	type DepositPerByte = DepositPerByte;
-	type CallStack = [pallet_contracts::Frame<Self>; 31];
+	type CallStack = [pallet_contracts::Frame<Self>; 5];
 	type WeightPrice = pallet_transaction_payment::Pallet<Self>;
 	type WeightInfo = pallet_contracts::weights::SubstrateWeight<Self>;
 	type ChainExtension = ();
 	type DeletionQueueDepth = DeletionQueueDepth;
 	type DeletionWeightLimit = DeletionWeightLimit;
 	type Schedule = Schedule;
 	type AddressGenerator = pallet_contracts::DefaultAddressGenerator;
-	type MaxCodeLen = ConstU32<{ 128 * 1024 }>;
+	type MaxCodeLen = ConstU32<{ 123 * 1024 }>;
```

Comment:
I'm wondering, shouldn't it be possible to calculate this based on the other config settings anyways, so we don't need to have the integrity check (ie. the code used in the integrity check could be used to calculate this). Unless there's a use case that requires to configure this value smaller than it has to be?

Reply:
I don't think we want this dynamic. Max code size is quite the weighty decision and should be done explicitly IMHO.

Reply:
Fair enough :+1: